api/src/main/java/org/openmrs/User.java

```diff
@@ -100,11 +103,37 @@
 	@Column(unique = true)
 	private String email;
 	
+	@Column(length = 50)
+	private String creator;
```

Comment:
So this should look like:

```java
@ManyToOne(optional = false)
@JoinColumn(name = "creator")
private User creator;
```

We want it to match the definition [here](https://github.com/openmrs/openmrs-core/blob/91dd1d3b555bd703a0086392a2f1d460353cc2c2/api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java#L42).

Basically, the point of the ticket is to remove the `name` and `description` fields that we aren't really using for User.